mne/tests/test_bem.py

```diff
@@ -173,10 +175,10 @@ def test_bem_solution():
 def test_fit_sphere_to_headshape():
     """Test fitting a sphere to digitization points"""
     # Create points of various kinds
-    rad = 90.  # mm
-    big_rad = 120.
-    center = np.array([0.5, -10., 40.])  # mm
-    dev_trans = np.array([0., -0.005, -10.])
+    rad = 0.09
+    big_rad = 0.12
+    center = np.array([0.005, -0.01, 0.04])
```

Comment:
If you look a few lines down, these values get added to the dig point
locations.  So it doesn't really matter what values they are, as long as
you get the same thing out at the end.